packages/kbn-pm/src/production/build_production_projects.ts

```diff
@@ -77,6 +77,17 @@ async function buildProject(project: Project) {
   }
 }
 
+/**
+ * Copy the all the project's files from it's "intermediate build directory" and
```

Comment:
__typo:__ `Copy the all the` --> `Copy all the`? 
__typo:__ `from it's` --> `from its`?

Reply:
:partyparrot: